src/style.css

```diff
@@ -1,3 +1,75 @@
+:root {
+  --primary-color: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.logo {
+  height: 40px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  width: auto;
+  background-color: white;
+  padding: 0 50px;
+  align-items: center;
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. You should provide a generic font family as a backup. For example, `font-family: Roboto, Arial, sans-serif;`.